### DISCUSSION TITLE
Installing LAMMPS built-in mode USER-DEEPMD package with CMAKE

### DISCUSSION THREAD

**Discussion by kimiaGF** [deepmodeling/deepmd-kit#2875] [Q&A]
I am trying to install LAMMPS with built-in `deepmd`, but I would prefer to use cmake (`make -D PKG_NAME=yes`) is there a way to do this? The documentation details installation with `make` but not `cmake`.

**Comment by njzjz** [comment]
You need to modify LAMMPS's CMake files. An example is given at https://github.com/conda-forge/lammps-feedstock/blob/294a97126e09e156f20cd61f3a952746e9cf303f/recipe/vcsgc_mtp_n2p2.patch

For how to Find DeePMD-kit from CMake, see https://docs.deepmodeling.com/projects/deepmd/en/master/development/cmake.html

**Comment by njzjz** [comment]
The documentation has been added in #2891.
